pkgs/servers/dante/default.nix

```diff
@@ -11,15 +11,16 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ pam libkrb5 cyrus_sasl miniupnpc ];
 
-  configureFlags = [
-    "--with-libc=libc.so.6"
-  ];
+  configureFlags = if stdenv.isDarwin then ["--with-libc=libc.dylib"]
```

Comment:
```suggestion
  configureFlags = ["--with-libc=libc${stdenv.targetPlatform.extensions.sharedLibrary}"]
```

Reply:
@LnL7 This change broke Dante under Linux. Dante doesn't work with the ld script `libc.so`, it needs the actual shared object (or a symlink to it) because it `dlopen()`s it at runtime. The method Dante's makefile uses to automatically locate the actual SO does not work within Nix's build environment, which is why I had the expression pointed to the correct file to begin with.

Reply:
I think I misunderstood what this was for then, sounds like it's probably not necessary on darwin at all in that case.